### PR TITLE
Use stricter yamllint config

### PR DESCRIPTION
**Pull request by cl8n**
seems more in line with what we end up editing so might as well

- [x] osu-wiki-tools update https://github.com/ppy/osu-wiki/pull/10523

should be easy to review one commit at a time. the largest changes are simple find+replace

SKIP_REMARK SKIP_OUTDATED_CHECK SKIP_WIKILINK_CHECK

### CODE REVIEW

**Review by TicClick — DISMISSED** [review]
lgtm for everything except `news/`

File: wiki/News_styling_criteria/en.md
Comment (Walavouchey) [inline wiki/News_styling_criteria/en.md:72]:
```suggestion
- `title` must represent the article's full title. Markdown formatting is disallowed in this string. News post titles, unlike wiki article titles and other headings, should be in title case. Note that surrounding quotes (`"`) may be necessary if the title contains colons (`:`) or number signs (`#`).
```

File: run-checks.sh
Comment (Walavouchey) [inline run-checks.sh:109]:
```suggestion
  _test_wrapper "YAML style" "_docker osu-wiki-tools check-yaml"
```

**Review by Walavouchey — COMMENTED** [review]
news look ok

File: run-checks.sh
Comment (cl8n) [reply run-checks.sh:109]: _test_wrapper wants a third "files" argument so it's written like this, I'll rewrite it better in a later pr.

File: .github/workflows/manual-sync.yml
Comment (peppy) [inline .github/workflows/manual-sync.yml:3]: I've never seen this style used in a yaml file for github workflows or otherwise. What's the deal.

Comment (cl8n) [reply .github/workflows/manual-sync.yml:3]: it's a weird gotcha that yaml keys can be parsed as other types and so this can be the boolean "true", which is why yamllint recommends quoting them like this. but checking again, I misunderstood which yaml parsers are used in our projects and it looks like this doesn't cause any problems so I'll change it back

**Review by Walavouchey — APPROVED** [review]

**Comment by peppy** [comment]
@Walavouchey in the future could you please not approve/merge when I've left a review on a PR? It's generally bad manners and you should wait for reapproval from all involved parties before going ahead with a merge.

**Comment by Walavouchey** [comment]
while i'll take that into account, we usually make this clear by self-requesting review or indicating in some way. i can't read your mind

**Comment by peppy** [comment]
I think you might be misunderstanding. I already left a review.

CleanShot 2023-11-24 at 04 12 54


You can't "re-request review" when you're asleep, and someone dismisses your review and approves/merges without consultation.

**Pull request merged**
